src/components/translation-builder/translation-builder.component.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import {
+  InlineLoading,
+  InlineNotification,
+  IconButton,
+  Tabs,
+  Tab,
+  TabList,
+  Dropdown,
+  ComposedModal,
+} from '@carbon/react';
+import { Download, Edit, ArrowRight } from '@carbon/react/icons';
+import { useLanguageOptions } from '@hooks/getLanguageOptionsFromSession';
+import EditTranslationModal from './edit-translation.modal';
+import { extractTranslatableStrings } from '../../utils/extractTranslatableStrings';
+
+import styles from './translation-builder.module.scss';
+interface TranslationBuilderProps {
+  formSchema: any;
+  onUpdateSchema: (updatedSchema: any) => void;
```

Comment:
Please avoid using `any` as the type, we should be able to use the type `Schema` here